pkg/app/server.go

```diff
@@ -89,6 +92,13 @@ func RunKubeStateMetricsWrapper(ctx context.Context, opts *options.Options) erro
 func RunKubeStateMetrics(ctx context.Context, opts *options.Options) error {
 	promLogger := promLogger{}
 	ksmMetricsRegistry := prometheus.NewRegistry()
+	exporter, err := otlpmetricgrpc.New(ctx, otlpmetricgrpc.WithInsecure())
```

Comment:
The below is all that is needed to export metrics from the ksmMetricsRegistry in OTLP format.